tools/cmake_test.py

```diff
@@ -234,7 +246,16 @@ def has_flag(flag, *synonyms):
                 args = [arg for arg in args if arg != "--"]
 
         elif "rpbench" in binary:
-            args = args + COMMON_TEST_ARGS
+            json_output = []
+            vbuild = find_vbuild_path_from_binary(self.binary)
+            bench_name = os.path.basename(self.binary)
+            if vbuild:
+                json_output = [
+                    "--json-output",
```

Comment:
Does this prevent the stdout output from also being printed?

Reply:
No, you still get the stdout output with this flag.